fluentd/configs.d/filter-pre-mux-client.conf

```diff
@@ -5,6 +5,8 @@
   shared_key "#{File.open('/etc/fluent/muxkeys/shared_key') do |f| f.readline end.rstrip}"
   secure yes
   ca_cert_path /etc/fluent/muxkeys/ca
+  buffer_queue_limit "#{ENV['BUFFER_QUEUE_LIMIT']}"
```

Comment:
Late review, but are these okay defaults?  If we have 1,024 * 16 MB chunks in the queue, how much memory does fluentd use then?  Isn't that 16 GB of memory?